src/pudl/helpers.py

```diff
@@ -36,6 +37,36 @@
 tz_finder = timezonefinder.TimezoneFinder()
 
 
+def download_zip_url(url, save_path, chunk_size=128):
+    """
+    Download and save a Zipfile locally.
+
+    Useful for acquiring and storing non-PUDL data locally.
+
+    Args:
+        url (str): The URL from which to download the Zipfile
+        save_path (pathlib.Path): The location to save the file.
+        chunk_size (int): Data chunk in bytes to use while downloading.
+
+    Returns:
+        None
+
+    """
+    # This is a temporary hack to avoid being filtered as a bot:
+    headers = {
+        'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:77.0) Gecko/20100101 Firefox/77.0',
+        'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8',
+        'Accept-Language': 'en-US,en;q=0.5',
+        'DNT': '1',
+        'Connection': 'keep-alive',
+        'Upgrade-Insecure-Requests': '1',
+    }
```

Comment:
These are fake headers which we can add to the request that Python is making to the FERC web server, that tricks FERC into thinking we are a normal person with a web browser, and not a script. Because they have started blocking things that look like scripts.